src/System Application/App/Performance Profiler/src/PerfProfilerScheduleCard.Page.al

```diff
@@ -223,10 +233,17 @@ page 1932 "Perf. Profiler Schedule Card"
         MaxRetentionPeriod: Duration;
         NoRetentionPolicySetupErr: Label 'No retention policy setup found for the performance profiles table.';
         CreateRetentionPolicySetupTxt: Label 'Create a retention policy setup';
+        NotEmptyDescriptionErr: Label 'The description must be filled in.';
```

Comment:
The variable name dis different from what the label is. It sounds like the description is not empty. When the label is a message saying that is must be filled in.
I would rename to something like `DescriptionNotFilledErr` or `EmptyDescriptionErr`.

Reply:
Fixed